src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ServerCertificates.cs

```diff
@@ -177,6 +201,7 @@ public static IEnumerable<object[]> UseCallback_ValidCertificate_ExpectedValuesD
             }
         }
 
+        [ActiveIssue(7812, TestPlatforms.Windows)]
```

Comment:
Since a new OS which fixes the problem can be either client or server side, yes, I can re-enable the test on newer Windows client builds.